framework/include/geomsearch/FindContactPoint.h

```diff
@@ -26,8 +26,9 @@ namespace Moose
 
 void
 findContactPoint(PenetrationInfo & p_info,
-                 FEBase * _fe,
-                 FEType & _fe_type,
+                 FEBase * fe_elem,
+                 FEBase * fe_side,
+                 FEType & fe_side_type,
```

Comment:
Renaming refactoring like this should ideally go in a separate commit, so it doesn't get mixed up with the actual fix you're making...  That being said, it's not too confusing in this case so I won't hold up the merge based on this comment ;0)